doc/server-secrets.md

```diff
@@ -218,7 +218,7 @@ installation access to private npm packages
 
 [npm token]: https://docs.npmjs.com/getting-started/working_with_tokens
 
-## Open Build Service
+### Open Build Service
```

Comment:
This isn't really related to this PR. I just noticed this heading level was wrong while I was editing the file.